src/main/java/org/assertj/core/api/ISoftAssertions.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * Copyright 2012-2020 the original author or authors.
+ */
+package org.assertj.core.api;
+
+public interface ISoftAssertions {
```

Comment:
maybe it is worth to add javadoc here

Reply:
`ISoftAssertions` is similar in concept and scope to `SoftAssertionsProvider`, which was merged in #1817. I think that at least this part of this PR is redundant.